rpcs3/Emu/System.cpp

```diff
@@ -372,6 +374,11 @@ std::string Emulator::GetBackgroundPicturePath() const
 
 bool Emulator::BootRsxCapture(const std::string& path)
 {
+	if (!IsStopped())
```

Comment:
this is actually just a placebo in both cases, as the emu is stopped before usage in every occasion anyway

Reply:
Even with the fix in #10836 I would like it to be forced by the function.

Reply:
it's also completely false, as it resets prior force_boot config.

Reply:
it should back up the force_boot and then SetForceBoot(tmp) after Stop

Reply:
I think it's about time to use real function arguments for Emulator::Stop instead of backing up this flawed logic don't you think?

Reply:
it's not really flawed because it's used to force boot games as well.

Reply:
and to be honest I don't have time to test this crap all over again

Reply:
Ok I've removed this commit from this pr.